src/main/resources/assets/securitycraft/lang/es_es.json

```diff
@@ -0,0 +1,767 @@
+	"block.securitycraft.reinforced_smooth_quartz_stairs": "Escaleras de cuarzo liso reforzadas",
+	"block.securitycraft.reinforced_granite_stairs": "Escaleras de granito reforzadas",
+	"block.securitycraft.reinforced_andesite_stairs": "Escaleras de andesita reforzadas",
+	"block.securitycraft.reinforced_red_nether_brick_stairs": "Escaleras de ladrillos del Nether rojos reforzadas",
+	"block.securitycraft.reinforced_polished_andesite_stairs": "Escaleras de andesita pulida reforzadas",
+	"block.securitycraft.reinforced_diorite_stairs": "Escaleras de diorita reforzadas",
+	"block.securitycraft.reinforced_polished_granite_slab": "Losa de granito pulido reforzada",
+	"block.securitycraft.reinforced_smooth_red_sandstone_slab": "Losa de arenisca roja lisa reforzada",
+	"block.securitycraft.reinforced_mossy_stone_brick_slab": "Losa de ladrillos de piedra musgosa reforzada",
+	"block.securitycraft.reinforced_polished_diorite_slab": "Losa de diorita pulida reforzada",
+	"block.securitycraft.reinforced_mossy_cobblestone_slab": "Losa de roca musgosa reforzada",
+	"block.securitycraft.reinforced_end_stone_brick_slab": "Losa de ladrillos de piedra del End reforzada",
+	"block.securitycraft.reinforced_smooth_sandstone_slab": "Losa de arenisca lisa reforzada",
+	"block.securitycraft.reinforced_smooth_quartz_slab": "Losa de cuarzo liso reforzada",
+	"block.securitycraft.reinforced_granite_slab": "Losa de granito reforzada",
+	"block.securitycraft.reinforced_andesite_slab": "Losa de andesita reforzada",
+	"block.securitycraft.reinforced_red_nether_brick_slab": "Losa de ladrillos del Nether rojos reforzada",
+	"block.securitycraft.reinforced_polished_andesite_slab": "Losa de andesita pulida reforzada",
+	"block.securitycraft.reinforced_diorite_slab": "Losa de diorita reforzada",
+	"block.securitycraft.reinforced_normal_stone_slab": "Losa de piedra reforzada",
+	"block.securitycraft.trophy_system": "Trofeo de torreta",
+	"block.securitycraft.crystal_quartz": "Bloque de cuarzo de cristal",
+	"block.securitycraft.chiseled_crystal_quartz": "Bloque de cuarzo de cristal cincelado",
+	"block.securitycraft.crystal_quartz_pillar": "Pilar de cuarzo de cristal",
+	"block.securitycraft.reinforced_crystal_quartz_block": "Bloque de cuarzo de cristal reforzado",
+	"block.securitycraft.reinforced_chiseled_crystal_quartz_block": "Bloque de cuarzo de cristal cincelado reforzado",
+	"block.securitycraft.reinforced_crystal_quartz_pillar": "Pilar de cuarzo de cristal reforzado",
+	"block.securitycraft.crystal_quartz_slab": "Losa de cuarzo de cristal",
+	"block.securitycraft.reinforced_crystal_quartz_slab": "Losa de cuarzo de cristal reforzado",
+	"block.securitycraft.crystal_quartz_stairs": "Escaleras de cuarzo de cristal",
+	"block.securitycraft.reinforced_crystal_quartz_stairs": "Escaleras de cuarzo de cristal reforzadas",
+	"block.securitycraft.block_pocket_wall": "Muro de bloque de bolsillo",
+	"block.securitycraft.block_pocket_manager": "Gestor de bloque de bolsillo",
```

Comment:
Sadly I don't. When he initially proposed the feature, he already called it Block Pocket, and during the months of development we talked about renaming it multiple times, but could never find a better name, so we decided to stick with it. The initial idea was to have blocks within the block pocket not render at all to bystanders, but that was thrown out of the window due to complexity.

Reply:
Really this is veeery hard. I share the feeling with @LorenaGdL, I translated it literally because I didn't want to leave it untranslated translating all the file. But I hope to find any better suggestion (the "Safe room" maybe looks something good for me).

Reply:
I'm not 100% on board with translating it to the equivalent of "Safe Room", as we would have called it that in the original localization should we have wanted that.

Reply:
Well, and «bloque secreto» (secret block, in English) could be an option? In the FTBWiki it is treated as a «secret chamber or secret entrance».

Reply:
Please, in the future don't make modifications to unresolved conversations so we can easily track what's pending.

Aside from that, my bad for not replying earlier about this issue. I talked to bl4ckscor3 and we reviewed how other languages were coping with this issue, and we decided that for now we will stick to the literal translation, even if it's not that descriptive. That means translations should be "Muro de bolsillo de bloques" and "Gestor de bolsillo de bloques"

Reply:
Okay! Sorry for making the modifications on my own in this string. I will change it for the literal translation in this new review :)